### ISSUE TITLE
affiliations not returning valid character info

### ISSUE THREAD

**Issue by jowrjowr** [ccpgames/esi-issues#527]
examples of characters:

```curl --compressed -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '[1575865391]' 'https://esi.tech.ccp.is/latest/characters/affiliation/?datasource=tranquility'```

or

```curl --compressed -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '[960322003]' 'https://esi.tech.ccp.is/latest/characters/affiliation/?datasource=tranquility'```

the return in both cases is:
```{"error": "No characters found!"}```

this is done direct through curl to eliminate any "environmental issues" (cache, *cough*)

which is obviously incorrect, especially since they work on the normal /characters/<charid>/ endpoint

i'd argue this in #esi directly but mods are being bunches of babyback bitches and banning me for a week over something trivial and stupid. sorry..

**Comment by aebe** [comment]
See here: https://developers.eveonline.com/blog/article/universe-names-update

particularly this paragraph:

> Second, there is a set of very old character and corporation ID's that exists in an extremely large legacy range. This range is approximately 2 billion ID's wide, which means that including it would nullify the protective value of this technique. Characters and corporations with IDs between 100000000 and 2099999999 should be resolved individually against the character and corporation endpoints until we can implement a more permanent solution.

tl;dr: known issue, it's going to stay that way for a while until the real fix is completed.

**Comment by tsuthers-ccpgames** [comment]
Yup, as per Aebe's reply, this is an unavoidable flaw in the compromise solution we're implementing to allow /universe/names/ to be brought back online. In the short term, IDs in that range will have to be resolved individually against the characters endpoint.
